packages/core/src/queries/user-sso-identities.ts

```diff
@@ -49,7 +49,7 @@ export default class UserSsoIdentityQueries extends SchemaQueries<
     detail: UserSsoIdentity['detail']
   ) {
     return buildUpdateWhereWithPool(this.pool)(this.schema, true)({
-      set: { detail },
+      set: { detail, updatedAt: Date.now() },
```

Comment:
is it possible to use database expression (`now()`)?

Reply:
maybe use this function

<img width="304" alt="image" src="https://github.com/user-attachments/assets/705c76f0-1ebd-49ee-a2e2-f13b10577dc0" />